packages/@aws-cdk/aws-ec2/lib/instance.ts

```diff
@@ -361,6 +412,59 @@ export class Instance extends Resource implements IInstance {
     this.role.addToPolicy(statement);
   }
 
+  /**
+   * Use a CloudFormation Init configuration at instance startup
+   *
+   * This does the following:
+   *
+   * - Attaches the CloudFormation Init metadata to the Instance resource.
+   * - Add commands to the instance UserData to run `cfn-init` and `cfn-signal`.
+   * - Update the instance's CreationPolicy to wait for the `cfn-signal` commands.
+   */
+  public applyCloudFormationInit(init: CloudFormationInit, options: ApplyCloudFormationInitOptions = {}) {
```

Comment:
Since this overrides a previously configured init, it kinda feels like it should only happen at instance creation time. Any specific reason its exposed post creation as well? Also, I might be missing something here, but wouldn't calling this twice actually produce a **single** `cfn-signal` call, while CloudFormation will be waiting for 2 signals per the creation policy?

> Side node: If we do keep this, maybe just `instance.init()`?